src/components/Filter/index.js

```diff
@@ -61,30 +62,32 @@ export default class Filter extends Component {
   };
 
   render() {
-    const { filterRenderer, data, dataKey, filterIconRenderer } = this.props;
+    const { filterRenderer, data, dataKey, filterTrigger } = this.props;
     return (
-      <div
-        className="Sticky-React-Table--Header-Column-Filter"
-        style={filtersStyles}
-      >
-        <div
-          className="Sticky-React-Table--Header-Column-Filter-Icon"
-          onClick={this.handleIconClick}
-          ref={this.handleIconRef}
-          style={filterIconStyles}
+      <Fragment>
+        <span
+          className="Sticky-React-Table--Header-Column-Filter"
+          style={filtersStyles}
         >
-          {renderElement(filterIconRenderer, {}, ':')}
-        </div>
-        {this.state.visible && (
-          <div
-            className="Sticky-React-Table--Header-Column-Filter-Dropdown"
-            ref={this.handleMenuRef}
-            style={filterDropdownStyles}
+          <span
+            className="Sticky-React-Table--Header-Column-Filter-Icon"
+            onClick={this.handleIconClick}
+            ref={this.handleIconRef}
+            style={filterIconStyles}
           >
-            {renderElement(filterRenderer, { data, dataKey }, null)}
-          </div>
-        )}
-      </div>
+            {renderElement(filterTrigger, {}, ':')}
+          </span>
+          {this.state.visible && (
+            <span
```

Comment:
Do not render Dropdown as a `span`. since we are providing a custom renderer, user won't be able to render divs inside of it. Dropdowns should always be a `div` because we do not know what the custom renderer will return.